src/keeper.ts

```diff
@@ -498,11 +584,23 @@ class Keeper {
     deps = { metricFuturesLiquidations: metrics.futuresLiquidations }
   ) {
     const taskLabel = "liquidation";
+    // check if it's liquidatable
     const canLiquidateOrder = await this.futuresMarket.canLiquidate(account);
     if (!canLiquidateOrder) {
-      this.logger.log("debug", `Cannot liquidate order`, {
-        component: `Keeper [${taskLabel}] id=${id}`,
-      });
+      // if it's not liquidatable update it's liquidation price
+      this.positions[account].liqPrice = parseFloat(
```

Comment:
Maybe it's slightly better if we run the liquidation check (and liquidate if possible) first. Then map over the events again and update the price?

Reply:
This code does try to liquidate first (it only checks the liquidation price if it can't liquidate). So it should cause no delays to actual liquidations. I was reluctant to add an additional flow for liquidation price updates, since I wanted to make sure the bot doesn't miss any positions that need updating.